app/components/search-facet-typeahead/component.js

```diff
@@ -49,8 +66,14 @@ export default Ember.Component.extend({
     },
 
     actions: {
-        changeFilter(selected) {
-            this.sendAction('onChange', this.get('key'), this.buildQueryObject(selected));
+        changeFilter(selected, match=true) {
```

Comment:
It's not used right now but it will be for adding things via clicking on search results.